test/passing/wrapping_functor_args.ml

```diff
@@ -0,0 +1,12 @@
+(* This declaration looks odd *)
+type request_token =
+  Sociaml_oauth_client.Client.Make(Sociaml_oauth_client.Posix.Clock)
+    (Sociaml_oauth_client.Posix.MAC_SHA1)
+    (Sociaml_oauth_client.Posix.Random).request_token
```

Comment:
This is clearly better than before, but I think `.request_token` ought to be on the next line since it selects from the functor application, not the last arg. Agree?

Reply:
I agree, but if I put a cut here it breaks some tests, I will try to fix it.